ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1394,27 +1393,29 @@ management_ssh:
       enable: < true | false >
 ```
 
-#### NTP Servers
-
-```yaml
-ntp_server:
-  local_interface:
-    vrf: < vrf_name >
-    interface: < source_interface >
-  nodes:
-    - < ntp_server_1 >
-    - < ntp_server_2 >
-```
-
 #### NTP
 
 ```yaml
 ntp:
+  local_interfaces:
+  - name: < source_interface >
```

Comment:
Structure allows to create more than one interface, while EOS only allows a single interface, last one is accepted.

Reply:
Ack. Fix on it's way.